tasks/section_6_System_Maintenance.yaml

```diff
@@ -143,9 +143,18 @@
 # 6.1.11 Ensure no unowned files or directories exist
 - name: 6.1.11 Ensure no unowned files or directories exist
   block:
-    - name: 6.1.11 Ensure no unowned files or directories exist | Find
-      shell: df --local -P | awk {'if (NR!=1) print $6'} | xargs -I '{}' find '{}' -xdev -nouser 2> /dev/null && true || true
-      register: output_6_1_11
+    - name: 6.1.11 Ensure no unowned files or directories exist | No skipped strings
+      block:
```

Comment:
I had to add blocks here because even if you skip a task, the value in `register: ` gets info about the task being skipped, so it would overwrite any variables with the same name.